spec/Appendix B -- Grammar Summary.md

```diff
@@ -413,3 +419,19 @@ TypeSystemDirectiveLocation : one of
 - `ENUM_VALUE`
 - `INPUT_OBJECT`
 - `INPUT_FIELD_DEFINITION`
+
+ServiceDefinition :
+
+- Description service [lookahead != `{`]
+- Description? service { ServiceAttribute+ }
+
+ServiceAttribute :
+
+- ServiceCapabilities
+
+ServiceCapabilities: capabilities { ServiceCapability+ }
```

Comment:
`*` instead of `+` and drop the group-less one?

I know historically we've done this "at least one required" grammatically instead of with validation + to do this extension-without-add pattern, but I think we've learned thats quite annoying in IDEs and tools. So I think we should explore dropping it here

```suggestion
ServiceDefinition :

- Description? service { ServiceAttribute* }

ServiceAttribute :

- ServiceCapabilities

ServiceCapabilities: capabilities { ServiceCapability* }
```

thoughts?